sesearch

```diff
@@ -143,7 +144,7 @@ else:
         warnings.simplefilter("ignore")
 
 try:
-    p = setools.SELinuxPolicy(args.policy)
+    p = setools.SELinuxPolicy(args.policy, gen_cil=args.gen_cil)
```

Comment:
Actually my first version used a similar method, the flag was stored as `PolicyObject::gen_cil`, set in sesearch near output stage. The problem I had with that approach was that a lot of effort was needed to make sure the flag was copied to all objects and that gets ugly. I never found out why the output in some case wasn't CIL. Also, `statement()` method is not used directly in a lot of places but `__str__()` instead. Perhaps just my poor Python skills are showing.

I'd even argue that while CIL output is indeed not a property of a policy, it's pretty much a global flag. There should never be a need for mixing CIL and kernel output.

But I'll try the `format()` method next, perhaps the format option gets passed around more easily,

Reply:
If I hand you a `policy.33` with no context whatsoever and ask you if it's source files were CIL, you can't answer because it's not a property of the policy.